client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -135,6 +135,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			this.props.updateFields( { [ name ]: ! this.props.fields[ name ] } );
 		};
 
+		handleAutosavingToggle = name => () => {
```

Comment:
It's your call really - I'm fine with both, but I think this suggestion might be the better way to go, as eventually we might need to introduce quite some more methods if we also auto-save other field types (see https://github.com/Automattic/wp-calypso/pull/11744#pullrequestreview-25244518 for more background on my point).